### DISCUSSION TITLE
Grammar Engineering meeting March 6 18:30 UTC

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#111] [Announcements]
Hope to see some of you at our biweekly meeting:

March 6 18:30 UTC

Zoom: 926 4912 4250
Password: 674288

https://cl-cam-ac-uk.zoom.us/j/92649124250?pwd=L2YzUE1pU3NHSmlsVDVnS203VmpNdz09 

Hosted kindly by @wsun106

**Comment by arademaker** [comment]
Hi @olzama , I believe @danflick and @fcbond will not be able to make next Monday. They will be both on their way to https://en.wikipedia.org/wiki/Machu_Picchu! Can we make March 13?

**Reply by olzama** [reply]
I think it is best to stick to the biweekly schedule, as some maybe have put it as a recurrent meeting on the calendar. We can try March 20 then? Unless it is starting to look like this schedule will no longer work for some of the core participants, in which case we should just pick a different one.

**Reply by arademaker** [reply]
Ok for me.

**Comment by olzama** [comment]
@wsun106 would you prefer to log in today, even if in the end it is a short meeting? Or would you prefer that we cancel today and just meet on March 20? I am totally fine either way.
